coordinator/src/utils/headers.ts

```diff
@@ -42,3 +43,8 @@ export function getScope(headers: Hapi.Util.Dictionary<string>): string {
   const defaultScope = `${PRESCRIBING_USER_SCOPE} ${DISPENSING_USER_SCOPE} ${TRACKER_USER_SCOPE}`
   return process.env.SANDBOX === "1" ? defaultScope : headers[RequestHeaders.SCOPE]
 }
+
+export function getOdsCode(headers: Hapi.Util.Dictionary<string>): string {
+  const defaultOdsCode = "FER21"
```

Comment:
probs should have a better default

Reply:
I see the default headers issue has been fixed, wondering if we want to add some tests to cover these, set `sandbox === "0"` and ensure there are no values set for these?